packages/two_dimensional_scrollables/test/table_view/table_test.dart

```diff
@@ -1365,10 +1365,11 @@ void main() {
 
     testWidgets('paint rects are correct when reversed and pinned',
         (WidgetTester tester) async {
+      // Regression test for https://github.com/flutter/flutter/issues/135386
```

Comment:
nit: want to maybe move this to line 1422 where the actual regression test for this begins to keep the context? The first few things in the test are unrelated, right?

Or maybe we should just have a file that contains all the screenshot tests as a separate test case and the whole file gets the comment that it should be deleted and migrated to mock_canvas once available? It seems like this text case is getting pretty large and I am afraid we will miss cases when we migrate the monolith over to mock_canvas...

Reply:
Oh I must've missed this in the merge. I added links to each issue this test was covering for that very reason to have references for the rewrite. Let me go find them again